protos/capture.proto

```diff
@@ -121,6 +121,16 @@ message CaptureEvent {
   }
 }
 
+message CaptureData {
+  map<uint64, uint64> function_count_map = 1;
+  map<uint64, Callstack> callstacks = 2;
```

Comment:
Weren't these already saved in the event buffer? I have a feeling we're carrying over the serialization of a lot of duplicated data.